source/fundamentals/serialization/guid-serialization.txt

```diff
@@ -24,87 +24,104 @@ In this guide, you can learn how to serialize **globally unique identifiers**
 (`GUIDs <https://learn.microsoft.com/en-us/dynamicsax-2012/developer/guids>`__),
 also known as **universally unique identifiers** (UUIDs).
 
-.. tip::
+A Short History of MongoDB GUIDs
```

Comment:
S: This subheading is a bit too "narrative" maybe something like

> Background on MongoDB GUIDs

or even just

> MongoDB GUIDs

Reply:
i just want our docs to be more fuuuuuuuuuuun